flask/app/participants.py

```diff
@@ -39,23 +39,31 @@
 @participants_blueprint.route("/api/participants", methods=["GET"])
 @login_required
 def list_participants() -> Response:
+
+    app.logger.debug("Parsing paged query parameters")
     # General paged query parameters
     # for some reason type=int doesn't catch non-integer queries, only returning None
+    app.logger.debug("Validating page")
     try:
         page = int(request.args.get("page", default=0))
         if page < 0:  # zero-indexed pages
             raise ValueError
     except:
+        app.logger.error("page must be a non-negative integer")
         abort(400, description="page must be a non-negative integer")
+    app.logger.debug("Validating limit...")
     try:
         limit = request.args.get("limit")
         if limit is not None:  # unspecified limit means return everything
             limit = int(limit)
             if limit <= 0:  # MySQL accepts 0 but that's just a waste of time
                 raise ValueError
     except:
+        app.logger.error("limit must be a positive integer")
         abort(400, description="limit must be a positive integer")
 
+    app.logger.debug("Parsing order query parameters")
+    app.logger.debug("Validating order_by")
     order_by = request.args.get("order_by", type=str)
```

Comment:
Could log the parsed 'order_by' argument to be in line with the rest of the query parameters?